storage/src/index/mod.rs

```diff
@@ -164,8 +165,8 @@ mod tests {
 
         // Values should be in stack order (last in first).
         assert_eq!(
-            index.get_iter(b"key").copied().collect::<Vec<_>>(),
-            vec![3, 2, 1]
+            index.get(b"key").copied().collect::<Vec<_>>(),
+            vec![1, 3, 2]
```

Comment:
Not that important but might there be a way to cleanly support inserting the latest element at the head instead of always in second position?

Reply:
Unfortunately, I couldn't find a way to do this without either making `V: Copy` or using `unsafe`. We'll have to live with it for now!